master/reference/calicoctl/resources/ippool.md

```diff
@@ -70,6 +70,19 @@ Increasing the block size from the default (e.g., using `24` for IPv4 to give 25
 
 Reducing the block size from the default (e.g., using `28` for IPv4 to give 16 addresses per block) means more blocks per host and therefore potentially more routes. This can be beneficial if it allows the blocks to be more fairly distributed amongst the hosts.
 
+#### Node Selector
+
+{% include {{page.version}}/selectors.md %}
+
+For details on configuring IP pool node selectors on your deployment, please read the
+[Assigning IP pools per-node guide]({{site.baseurl}}/{{page.version}}/usage/assigning-ip-pools-per-node).
+
+> **Note**: The pool's `disabled` field takes higher precedence than `nodeSelector`
+> i.e. {{site.prodname}} IPAM will not allocate any IPs from the pool if disabled even if it selects the node that it is on.
+> Additionally, any pools that are [specified through the IPAM config]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration#ipam)
```

Comment:
It's not just the IPAM config - it could also be one of the annotations we support on pods / namespaces. 

Which reminds me, we should probably adjust some of the content here to mention the interactions with selectors: https://deploy-preview-2371--calico.netlify.com/master/reference/cni-plugin/configuration